polars/polars-core/src/serde/series.rs

```diff
@@ -154,7 +151,9 @@ impl<'de> Deserialize<'de> for Series {
                     }
                     DataType::UInt32 => {
                         let values: Vec<Option<u32>> = map.next_value()?;
-                        Ok(Series::new(&name, values))
+                        let mut ca = UInt32Chunked::new(&name,values);
```

Comment:
If we add `get_flags` on the `Series` we can serialize the flags in the generic part of the code, meaning that we don't have to repeat for every type.